src/core/Akka.Remote/RemoteWatcher.cs

```diff
@@ -498,7 +498,7 @@ private void SendHeartbeat()
 
         private void TriggerFirstHeartbeat(Address address)
         {
-            if (WatchingNodes.Contains(address) && !_failureDetector.IsMonitoring(address))
+            if (WatcheeByNodes.ContainsKey(address) && !_failureDetector.IsMonitoring(address))
```

Comment:
`WatchingNodes` is an alias for the keys collection for `WatcheeByNodes`, so this code effectively does the same thing that it did before. Difference is that it reads exactly like the JVM's now.